src/linagora.esn.calendar/app/routes.js

```diff
@@ -32,6 +32,24 @@ function routesConfig($stateProvider) {
         }
       }
     })
+    .state('calendar.main.participation', {
+      url: '/participation',
+      resolve: {
+        event: function($log, $state, $location, notificationFactory, calEventService) {
+          const jwt = $location.search().jwt;
+          const eventUid = $location.search().eventUid;
+
+          if (jwt && eventUid) {
+            calEventService.changeParticipationFromLink(eventUid, jwt)
+              .catch(err => {
+                $log.error('Can not display the requested event', err);
+                notificationFactory.weakError(null, 'Can not display the event');
```

Comment:
Is there translation (i18n) for this?

Reply:
yes , i picked it from translation files